content/platforms/kubernetes/kubernetes-persistent-volumes.md

```diff
@@ -1,6 +1,7 @@
 ---
-Title: Deploying Kubernetes with Persistent Volumes in Operator-based Architecture
-description: 
+Title: Deploying with Persistent Volumes
+description: A Redis Enterprise Cluster deployed with the operator should include
```

Comment:
@alexmilowski this is somewhat misleading since we implicitly do that even if it isn't specified in the spec. On a different note, I would stick with a description of the article rather than providing directives in the `Description` field.

Reply:
"Configuring Persistent Volumes" ?

The descriptions are autogenerated by taking the first few sentences from the markdown. It can generate nonsense quite easily. Summaries need to be written and not generated.

Reply:
How are descriptions auto-generated? is that a new feature? can we disable it? I thought you wrote in your PR that you "added descriptions"?

Reply:
They just pull some text from the targeted file ... the first few sentences excluding any section titles.